File: Ejercicio6.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ejercicio6
{
    internal class Program
    {
        static void Main(string[] args)
        {
            double a, b, x, igv, total;
            Console.Write("Ingresa el valor de a: ");
            a = double.Parse(Console.ReadLine());
            Console.Write("Ingresa el valor de b: ");
            b = double.Parse(Console.ReadLine());
            x = Math.Pow(a + b, 2);
            igv = x * 0.19;
            total = x + igv;
            Console.WriteLine("IGV (19%): " + igv);
            Console.WriteLine("Precio total con IGV: " + total);
        }
    }
}

File: Ejercicio6.java
import java.util.Scanner;

public class Ejercicio6 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Ingrese a: ");
        double a = sc.nextDouble();
        System.out.print("Ingrese b: ");
        double b = sc.nextDouble();

        double x = Math.pow((a + b), 2);
        double igv = x * 0.19;
        double total = x + igv;

        System.out.println("Valor sin IGV: " + x);
        System.out.println("IGV (19%): " + igv);
        System.out.println("Total con IGV: " + total);
    }
}
